META/sqisign_lvl1.yml

```diff
@@ -0,0 +1,63 @@
+name: SQIsign_lvl1
+type: signature
+claimed-nist-level: 1
+length-public-key: 65
+length-secret-key: 353
+length-signature: 148
+nistkat-sha256: cce2e6a0e6aff1179eba6b8b5cb1b096e751f9aafdf8934a2df857164a902202
+principal-submitters:
+  - Marius A. Aardal
+  - Gora Adj
+  - DiegoF.Aranha
```

Comment:
Here is another name that is messed up.  I assume Tako BorrisFouotsa is also wrong.  There seem to still be a few typos here.